src/main/java/housing/Region.java

```diff
@@ -84,6 +130,51 @@ public void step() {
         // Stores regional household statistics after both regional markets have been cleared
         regionalHouseholdStats.record();
     }
+    
+    //******** Methods for Failed Bidders ******//   
```

Comment:
**General comment for this section**: Bfffff, are we sure this is the easiest way of doing this? I really think that counting failed bids as a variable in each household would be easier... Could you convince me that this way is better or more effective in any fundamental way?

Reply:
Yes, I agree with you on this. I will count them in the Household instead.

Reply:
Great, thanks!